ckan/templates/macros/form.html

```diff
@@ -79,7 +79,17 @@
     Examples:
 
     {% import 'macros/form.html' as form %}
-    {{ form.select('year', label=_('Year'), options=[{'name':2010, 'value': 2010},{'name': 2011, 'value': 2011}], selected=2011, error=errors.year) }}
+    {{ form.select('year', label=_('Year'), options=[{'value':2010, 'text': 2010},{'value': 2011, 'text': 2011}], selected=2011, error=errors.year) }}
+
+    Or only with values if they are the same as text:
+    {{ form.select('year', label=_('Year'), options=[{'value':2010},{'value': 2011}], selected=2011, error=errors.year) }}
+
+    Complete example:
+    {{ form.select('the_data_type', id='the_data_type', label=_('Data Type'), options=[
+        {'value': '0', 'text': _('[Choose Data Type]')}
+        , {'value': '1', 'text': _('Private data')}
+        , {'value': '2', 'text': _('Not private data')}
+    ], selected=data.the_data_type if data.the_data_type else '0', is_required=true) }}
```

Comment:
Ignore this file in this PR. I did a rebase and included this commit by mistake. The commit id is unchanged, but git thinks I coauthored it now... This [commit](https://github.com/ckan/ckan/pull/5304/commits/8fee4e8fbf2b81ad6c4c48470c823abf03f1edf6) is already merged to master 🤷‍♂